ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -0,0 +1,634 @@
+    Click this entry to create a view. The *Create view* window is displayed.
+
+    - ![Rename](/Assets/Icons/Edit02.png "[Rename]") rename  
+    Click this entry to rename the selected view. The *Rename view* window is displayed. This menu entry is only displayed when a view is selected.
+
+    - ![Reset](/Assets/Icons/Reset.png "[Reset]") reset  
+    Click this entry to reset the view to the selected views settings. This menu entry is only displayed when a view is selected and any changes are made to the views settings.
+
+    - ![Publish](/Assets/Icons/Publish.png "[Publish]") publish  
+    Click this entry to publish the view. This menu entry is only displayed when a view is selected and unpublished.
+
+    - ![Unpublish](/Assets/Icons/Unpublish.png "[Unpublish]") unpublish  
+    Click this entry to unpublish the view. This menu entry is only displayed when a view is selected and published.
+
+    - ![Save](/Assets/Icons/Save.png "[Save]") save  
+    Click this entry to save the current view settings in the selected view. This menu entry is only displayed when a view is selected.
+
+     > [Info] When the settings of a view have been changed, an asterisk is displayed behind the view name. The asterisk is hidden as soon as the changes have been saved.
+
+    - ![Delete](/Assets/Icons/Trash01.png "[Delete]") delete  
+    Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed when a view is selected.
+
+
+- *Log Level*  
+Click the drop-down list to select a log level. The following options are available:  
+
+    - **All**  
+    Select this option to display all logs, regardless of their status.
+
+    - **Debug**
+
+    - **Info**  
+    Select this option to display all logs simply providing information.
+
+    - **Notice**
+
+    - **Warning**
+
+    - **Error**  
+    Select this option to display all logs specifying an error.
+
+    - **Critical**
+
+    - **Alert**
+
+    - **Emergency**
+
+[comment]: <> (Log status durchgehen?)
+
+
+- ![Search](/Assets/Icons/Search.png "[Search]") (Search)  
+Click this button to display the search bar and search for a workflow.
+
+- ![Refresh](/Assets/Icons/Refresh01.png "[Refresh]") (Refresh)  
+Click this button to update the list of workflows.
+
+- ![Columns](/Assets/Icons/Columns.png "[Columns]") Columns (x)  
+Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The x indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](/Assets/Icons/Filter.png "[Filter]") Filter (x)  
+Click this button to display the filter bar and customize the active filters. The x indicates the number of filters that are currently active.
+
+
+The list displays all logs. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+  - [x]    
+  Select the checkbox to display the editing toolbar.
```

Comment:
Oli: what is the toolbar good for? No displayed buttons...

Reply:
Kommentar hinzugefügt